src/Microsoft.AspNetCore.Mvc.ViewFeatures/PageRemoteAttribute.cs

```diff
@@ -10,60 +10,29 @@
 namespace Microsoft.AspNetCore.Mvc
 {
     /// <summary>
-    /// A <see cref="RemoteAttribute"/> for razor page handler which configures Unobtrusive validation 
+    /// A <see cref="RemoteAttributeBase"/> for razor page handler which configures Unobtrusive validation 
     /// to send an Ajax request to the web site. The invoked handler should return JSON indicating 
     /// whether the value is valid.
     /// </summary>
     /// <remarks>Does no server-side validation of the final form submission.</remarks>
-    public class PageRemoteAttribute : RemoteAttribute
+    [AttributeUsage(AttributeTargets.Property, AllowMultiple = false, Inherited = true)]
+    public class PageRemoteAttribute : RemoteAttributeBase
     {
-        private readonly string _pageHandler;
-        private readonly string _pageName;
-
         /// <summary>
-        /// Initializes a new instance of the <see cref="PageRemoteAttribute"/> class.
+        /// The handler name used when generating the URL where client should send a validation request.
         /// </summary>
         /// <remarks>
-        /// Will use ambient page name and handler name when generating the URL where client
-        /// should send a validation request.
+        /// If not set the ambient value will be used when generating the URL.
         /// </remarks>
-        public PageRemoteAttribute()
-            : this(pageHandler: null)
-        {
-
-        }
+        public string PageHandler { get; set; }
```

Comment:
I went with @pranavkm suggestion here and implemented public properties instead of having multiple constructors.